src/cli.rs

```diff
@@ -81,6 +84,10 @@ impl Options {
                 .short("t")
                 .default_value(DEFAULT_TITLE)
                 .help("Defines the window title"))
+            .arg(Arg::with_name("class")
+                 .long("class")
+                 .default_value(DEFAULT_CLASS)
```

Comment:
Sorry that I have to nitpick again, but I don't think `-c` is currently used by anything. So adding the `.short("c")` similar to the `.short("t")` for title would probably be a good addition.

Reply:
It's not currently used, but I'd imagine the `--class` flag will likely be used rarely and mostly in scripts, so there's no harm in writing the long option. And in case a more important flag gets added it won't break anything.

Reply:
Yeah I had a similar train of thought, especially because class is only supported on one platform.

It's probably a better idea to use `-c` for the already existing `--config-file`, thanks for speaking up. :)